.env.example

```diff
@@ -15,4 +15,4 @@ TWITTER_APP_SECRET=
 
 GOOGLE_ID=
 GOOGLE_SECRET=
-SECRET_KEY =
```

Comment:
this SECRET_KEY should not be removed cause it is used by the JWT
and SECRET is for social login